examples/example-libevent-ssl.c

```diff
@@ -50,14 +50,15 @@ int main (int argc, char **argv) {
     const char *cert = argv[4];
     const char *certKey = argv[5];
     const char *caCert = argc > 5 ? argv[6] : NULL;
+    enum redisSslMode sslMode = HIREDIS_SSL_VERIFY_PEER;
```

Comment:
Use a typedef

Reply:
I'd actually prefer that this was a bitmask of options (and therefore a `#define`); this way we can semantically extend it later on if we need to pass other flags.

Maybe we can even do this with a `redisSslOptions` struct for future use?